maat-court-data-api/src/main/java/gov/uk/courtdata/billing/controller/ApplicantResetController.java

```diff
@@ -0,0 +1,33 @@
+package gov.uk.courtdata.billing.controller;
+
+import gov.uk.courtdata.billing.request.ApplicantResetRequest;
+import gov.uk.courtdata.billing.service.ApplicantResetService;
+import gov.uk.courtdata.eform.controller.StandardApiResponseCodes;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Tag(name = "Applicant CCLF Reset", description = "Rest API for resetting SEND_TO_CCLF flag in applicants")
+@RequestMapping("${api-endpoints.billing-domain}/applicant")
+public class ApplicantResetController {
+
+    private final ApplicantResetService applicantResetService;
+
+    @PatchMapping("/reset")
```

Comment:
If you can change the `BillingApplicantController`'s `@RequestMapping` on line 21 to be `@RequestMapping("${api-endpoints.billing-domain}/applicants")` then I think you can get rid of the `"/reset"` mapping here (you'll also need to remove `"/applicants"` on line 26 of the controller for existing tests to pass).